src/omero_zarr/cli.py

```diff
@@ -278,6 +278,15 @@ def _configure(self, parser: Parser) -> None:
             default=None,
             help="Maximum number of workers (only for use with bioformats2raw)",
         )
+        export.add_argument(
+            "--name_by",
+            default="id",
+            choices=["id", "name"],
+            help=(
+                "How to name the Image or Plate zarr. Default 'id' is [ID].zarr. "
```

Comment:
That raises the question about the extension.
In both cases, the library is used to generate ``ome.zarr`` so in one case using ``.zarr`` and in the other case ``.ome.zarr`` is confusing

Reply:
In general I think we've stated elsewhere:
 - *strongly* SHOULD (bordering on MUST): `.zarr`
 - SHOULD: `.ome.zarr`

Reply:
Breaking in the sense that you think current processes won't detect that something has already been exported and will re-export?

Reply:
No, just if you've got a workflow like:
```
$ omero zarr export Image:1
$ mv 1.zarr renamed_image.zarr
```
that workflow will need to change because `1.zarr` will now be created as `1.ome.zarr`

But that's not a major concern, so happy to make that change.

Reply:
:+1: I'm personally a bit up in the air. A benefit of `.zarr` filesets are that other stuff can be stored in them which is why I don't think we should ever _require_ `.ome.zarr`. On the flip side, what we do now will likely be the model that other submitters to BIA tend to follow, so it does make sense to do what we think is most strategic (and update any existing workflows as necessary). All that being said, :100: for doing whatever consistently.